src/compute/src/render/mod.rs

```diff
@@ -100,6 +100,7 @@
 //! stream. This reduces the amount of recomputation that must be performed
 //! if/when the errors are retracted.
 
+use itertools::izip;
```

Comment:
Nit: This should be in the third-party import block below.

Reply:
The import block below is DD/Timely. I created a new import block between std and the DD/Timely block. Is this what you meant?

Reply:
Normally (though I don't think that's spelled out anywhere), we structure our imports into blocks like this:

* `std` imports
* third-party imports
* `mz_` imports
* `crate` imports
* `super` imports
* `self` imports

(The last three blocks I'm actually not sure about. I think they are sometimes merged into one.)

According to this, DD, Timely, and `itertools` all should be in the third-party import block. I don't think we usually do dedicated import blocks for DD/Timely. In this case it just looks like we do because those are the only third-party crates we use in this module.

Reply:
Ok, fixed, thanks!